src/amo/components/SearchSort/SearchSort.scss

```diff
@@ -0,0 +1,82 @@
+@import '~core/css/inc/mixins';
+@import '~core/css/inc/vars';
+
+@mixin arrow-link-right-aligned() {
+  background-image: url('~amo/img/icons/arrow-link-color.svg');
+  margin-right: 0;
+  position: absolute;
+  right: 8px;
+  top: 14px;
+}
+
+.SearchSort {
+  background: $base-color;
+  border-radius: $border-radius-default;
+  border: 0;
+  margin: 0 5% 10px;
+  overflow: hidden;
+}
+
+.SearchSort-toggle {
```

Comment:
It was really convenient to have that `Card` component ready to use but I don't think we have any other uses for this one lined up other than "it looks like something we'll reuse" so that's cool.